Framework/Core/test/test_DataAllocator.cxx

```diff
@@ -143,8 +143,10 @@ DataProcessorSpec getSinkSpec()
       dumpStack(dh);
     }
     // plain, unserialized object in input1 channel
+    LOG(INFO) << "extracting o2::test::TriviallyCopyable from input1";
```

Comment:
I think tests should not do printouts, actually. Why do you need those?

Reply:
it turned out to be convenient to know at which operation an exception was thrown. Maybe we should extend the information which is printed out at an unhandled exception.

Reply:
I think you can better do that using [BOOST_TEST_CHECKPOINT](https://www.boost.org/doc/libs/1_69_0/libs/test/doc/html/boost_test/test_output/test_tools_support_for_logging/checkpoints.html).

Reply:
In principle much better, but the unit test is a workflow executable, I will check separately if the boost unit test macros can be used without the boost unit test driver.

Reply:
yeah, agreed. I did not notice it's a Device. No worries then.